addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/WappalyzerJsonParser.java

```diff
@@ -280,7 +280,7 @@ private static ImageIcon createSvgIcon(URL url) {
                 }
             }
         } else if (json != null) {
-            logger.error(
+            logger.debug(
```

Comment:
This one should be kept as is, it's unrelated. The log message could be tweaked to be more clear though, e.g.:
```Java
"Unexpected JSON type for {} pattern: {} {}", type, json, json.getClass().getCanonicalName())
```
(Could be applied to the other message too.)